modules/@apostrophecms/page/index.js

```diff
@@ -142,37 +139,6 @@ module.exports = {
           return result;
         }
 
-        // If I can't edit at least one of a node's
```

Comment:
Why does this no longer matter? Because page editing isn't going to be individually permitted?

Reply:
Not in core, correct.